bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/Manifest.cs

```diff
@@ -1,1378 +1,1339 @@
-    using NativeLanguage = NativeInterface.Language.LanguageHandle;
-    using NativeInternationalizedText = NativeInterface.InternationalizedText.InternationalizedTextHandle;
-    using NativeContactInformation = NativeInterface.ContactInformation.ContactInformationHandle;
-    using NativeGeopoliticalUnit = NativeInterface.GeopoliticalUnit.GeopoliticalUnitHandle;
-    using NativeBallotStyle = NativeInterface.BallotStyle.BallotStyleHandle;
-    using NativeParty = NativeInterface.Party.PartyHandle;
-    using NativeCandidate = NativeInterface.Candidate.CandidateHandle;
-    using NativeSelectionDescription = NativeInterface.SelectionDescription.SelectionDescriptionHandle;
-    using NativeContestDescription = NativeInterface.ContestDescription.ContestDescriptionHandle;
-    using NativeContestDescriptionWithPlaceholders = NativeInterface.ContestDescriptionWithPlaceholders.ContestDescriptionWithPlaceholdersHandle;
-    using NativeManifest = NativeInterface.Manifest.ManifestHandle;
-    using NativeInternalManifest = NativeInterface.InternalManifest.InternalManifestHandle;
-    using NativeElementModP = NativeInterface.ElementModP.ElementModPHandle;
-    using NativeElementModQ = NativeInterface.ElementModQ.ElementModQHandle;
-
-    #region AnnotatedString
-
-    /// <Summary>
-    /// Use this as a type for character strings.
-    /// See: https://developers.google.com/elections-data/reference/annotated-string
-    /// </Summary>
-    public class AnnotatedString : DisposableBase
-    {
-        /// <Summary>
-        /// An annotation of up to 16 characters that's associated with a character string.
-        /// </Summary>
-        public unsafe string Annotation
-        {
-            get
-            {
-                var status = NativeInterface.AnnotatedString.GetAnnotation(
-                    Handle, out IntPtr value);
-                if (status != Status.ELECTIONGUARD_STATUS_SUCCESS)
-                {
-                    Console.WriteLine($"AnnotatedString Error Annotation: {status}");
-                    return null;
-                }
-                return Marshal.PtrToStringAnsi(value);
-            }
-        }
-
-        /// <Summary>
-        /// The character string
-        /// </Summary>
-        public unsafe string Value
-        {
-            get
-            {
-                var status = NativeInterface.AnnotatedString.GetValue(
-                    Handle, out IntPtr value);
-                if (status != Status.ELECTIONGUARD_STATUS_SUCCESS)
-                {
-                    Console.WriteLine($"AnnotatedString Error Value: {status}");
-                    return null;
-                }
-                return Marshal.PtrToStringAnsi(value);
-            }
-        }
-
-        internal unsafe NativeAnnotatedString Handle;
-
-        unsafe internal AnnotatedString(
-            NativeAnnotatedString handle)
-        {
-            Handle = handle;
-        }
-
+﻿using System;

```

Comment:
Is this white space changes throughout this file?

Reply:
I'm noticing the white spacing issues in the repo. Perhaps this should be a setting with a configuration to avoid this going forward. Maybe a separate issue.

Reply:
Probably separate issue to fix for real, added this ticket: https://github.com/microsoft/electionguard-cpp/issues/249